02-Reproject-Raster-In-R.Rmd

```diff
@@ -246,15 +258,16 @@ plot(DTM_HARV,
      legend=F)
 ```
 
-We have now successfully layered the Digital Terrain Model on top of our hillshade
-to produce a nice looking, textured basemap! 
+We have now successfully layered the Digital Terrain Model on top of our
+hillshade to produce a nice looking, textured base map! 
+
+##Challenge
```

Comment:
I started using > to format challenges. 

We have a few options. We could also create a DIV tag that allows markdown. That could be the best option as it's fewer lines of code. We should start an issue on this in our style template. Or here and move it to the style template for final implementation. 

Essentially it seems you have some things you like and i have some i like and we will go back and forth on preferences unless we create an issue and define final style applications